codec/testdata/animal.go

```diff
@@ -0,0 +1,25 @@
+package testdata
```

Comment:
will these be removed?

Reply:
no. it's used for tests in the `codec` pkg